blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,204 @@
+
+
+Let’s add #Cases to Rows and it draws a graph!
+![](https://i.imgur.com/NrJmp4B.jpg)
+
+Now, let's apply date range filter.
+Add Date column to Filters and select “Range of Dates” and the range of dates automatically shows the range from start date to end date. Hit OK.
+
+![](https://i.imgur.com/TvTq1F7.jpg)
+
+![](https://i.imgur.com/nOSmHtM.jpg)
+
+Click the Date filter and select “Show filter.”
+Here we go! The date filter appears on the right side of the dashboard.
+
+![](https://i.imgur.com/GPmQW1X.jpg)
+
+![](https://i.imgur.com/9qxEOry.jpg)
+
+As mentioned above, we would like to display #Cases by state. Drag state dimension to Marks >> Color
+
+![](https://i.imgur.com/EtVsmWf.jpg)
+
+This warning pops up because the dataset contains 54 states, so we will apply a filter to select top states. Click “Filter and then add.” But if you would like to see all states, click “Add all members."
+
+![](https://i.imgur.com/KUUBE5R.jpg)
+
+Select California, Illinois, New Jersey, and New York, which are the states with most coronavirus cases.
+
+![](https://i.imgur.com/osQETRU.jpg)
+
+Tada!
+The graph below displays COVID-19 cases by state.
+![](https://i.imgur.com/0HDFtwb.jpg)
+
+If you change the date range, the bar graph updates accordingly.
+![](https://i.imgur.com/schpEg4.jpg)
+
+If you would like to apply date range filter to sheet 1 (COVID-19 case by state and County), you can do so by applying the date filter to all worksheets using the same data source. Now, the date filter will be applied to sheet 1 and sheet 2.
+
+![](https://i.imgur.com/3XZuNIM.jpg)
+
+Step 4: Format the dashboard
+- 
+
+In dashboard 1, drag sheet 1 to the blank page.
+
+![](https://i.imgur.com/ORdHKDE.jpg)
+
+Then you will notice that this dashboard does not have any space to display the bar graph we created in sheet 2. We should expand the size of the dashboard. 
+
+The default size is 1000 x 800. Let’s increase the height to 1600.
+
+![](https://i.imgur.com/iLXt34y.jpg)
+
+
+Select sheet 1 and click the Downward Arrow and select Floating. This allows you to freely move the table and resize the table.
+
+![](https://i.imgur.com/K25QOae.jpg)
+
+Click “Floating” on State and County filter and move around the table and two filters as you would like.
+![](https://i.imgur.com/eAg9Add.jpg)
+
+Follow the same steps above and add sheet 2 to the dashboard.
+
+![](https://i.imgur.com/NGeWYvY.jpg)
+
+Now, let's add title to the dashboard!
+
+Check "Show dashboard title" in the Objects section and add the title.
+
+![](https://i.imgur.com/hbZbZ1h.jpg)
+
+The dashboard looks too plain, so let's add a background color to the title.
+
+![](https://i.imgur.com/8Er750I.jpg)
+
+Right click the title and click "Format title…"
+
+![](https://i.imgur.com/YDKw8AU.jpg)
+
+Apply Shading in the Dashboard Title section.
+
+![](https://i.imgur.com/RsmLOhP.jpg)
+
+
+Here is the final product of the dashboard!
+
+![](https://i.imgur.com/dbwHZVE.jpg)
+![](https://i.imgur.com/4Y8RT3m.jpg)
+
+
```

Comment:
I would add a conclusion here and hit two points:
1: what next steps are or which other features you would recommend looking into next
2: a link to your dashboard if it'd be possible to make it public but not let people edit it